### PR TITLE
If youtube atom is embed then set origin on iframe to externalEmbedHost

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

- If youtube atom is embed then set origin on iframe to externalEmbedHost
- embedPage is set to false for media.scala.html and true for mediaEmbed.scala.html

## What is the value of this and can you measure success?

Embedded youtube atoms will be enhanced by JavaScript.

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

### CODE REVIEW

File: common/app/views/fragments/atoms/youtube.scala.html [hunk 1]
```diff
@@ -1,15 +1,16 @@
 @import conf.Configuration.site.host
+@import conf.Configuration.Media
```
Comment (gidsg) [inline common/app/views/fragments/atoms/youtube.scala.html]: I think this import could be combined with the one above

File: common/app/views/fragments/atoms/media.scala.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@(media: _root_.model.content.MediaAtom, amp: Boolean = false, displayCaption: Boolean)(implicit request: RequestHeader, context: model.ApplicationContext)
+@(media: _root_.model.content.MediaAtom, amp: Boolean = false, displayCaption: Boolean, embedPage: Boolean = false)(implicit request: RequestHeader, context: model.ApplicationContext)
```
Comment (TBonnin) [inline common/app/views/fragments/atoms/media.scala.html]: There are a lot of booleans passed to the this template 😜

**Review by TBonnin — APPROVED** [review]

**Comment by Aliihi** [comment]
yhyaalmzory4

**Comment by gidsg** [comment]
👍 🔖

**Pull request merged**